### PR TITLE
chore(ts): Convert accountEmails

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  emails: UserEmail[];
+};
+
+class AccountEmails extends AsyncView<Props, State> {
+  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
+    return [['emails', ENDPOINT]];
+  }
+
+  getTitle() {
+    return 'Emails';
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx:35]:
```suggestion
    return t('Emails');
```

File: src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    });
+
+  renderBody() {
+    const {emails} = this.state;
+    const primary = emails?.find(({isPrimary}) => isPrimary);
+    const secondary = emails?.filter(({isPrimary}) => !isPrimary);
+
+    return (
+      <div>
+        <SettingsPageHeader title="Emails" />
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx:87]:
```suggestion
        <SettingsPageHeader title={t('Emails')} />
```

File: src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        </Button>
+      )}
+      {!hideRemove && !isPrimary && (
+        <Button
+          data-test-id="remove"
+          priority="danger"
+          size="small"
+          icon={<IconDelete />}
+          onClick={e => onRemove(email, e)}
+        />
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx:175]: Should we add aria label?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx:175]: yeah I can

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/account/accountEmails.tsx:175]: I'll do it right after this PR actually

**Review by scttcper — APPROVED** [review]

**Pull request merged**